redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -56,42 +56,42 @@ DatabaseServerV4Responses list(@NotNull @QueryParam("environmentId") String envi
     DatabaseServerV4Response get(@NotNull @QueryParam("environmentId") String environmentId, @PathParam("name") String name);
 
     @POST
-    @Path("external")
+    @Path("/managed")
```

Comment:
what does it mean: `managed`? Do you have any other _type_? Why do you want to add this to the URL?